tests/ui/imports/glob-conflict-cross-crate.rs

```diff
@@ -1,8 +1,13 @@
+// check-pass
 // aux-build:glob-conflict.rs
 
 extern crate glob_conflict;
 
 fn main() {
-    glob_conflict::f(); //~ ERROR cannot find function `f` in crate `glob_conflict`
```

Comment:
These errors should stay errors.
Only `warn_ambiguity == true` cases should be demoted to warnings.